src/domains/ecommerce-app/api/ecommerce-io/v1/_auth_request.xml.tpl

```diff
@@ -2,7 +2,6 @@
     <inbound>
         <base />
         <set-header name="x-pgs-id" exists-action="delete" />
-        <set-header name="x-user-id" exists-action="delete" />
```

Comment:
Why is it deleted? this header must be set by the policy from sessionToken (only for ecommerce flow with jwt), and then passing it to the underlying service

Reply:
This field is set in the base policy. If you remove the header here it will result in 404 since now x-user-id is used to check the user of the transactions.